{{cookiecutter.repo_name}}/.gitignore

```diff
@@ -92,3 +92,9 @@ docs/_build/
 
 # PyBuilder
 target/
+Dockerfile
```

Comment:
Just to keep a clean discussion history.

`docker-base` in the parent dir of the project is deprecated. We no longer want to have dangling directories like that.

Everything that is build related will live under `{{ project_name }}.build`.